sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/DefaultAggregation.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.view;
+
+import io.opentelemetry.sdk.common.InstrumentationLibraryInfo;
+import io.opentelemetry.sdk.internal.ThrottlingLogger;
+import io.opentelemetry.sdk.metrics.common.InstrumentDescriptor;
+import io.opentelemetry.sdk.metrics.exemplar.ExemplarFilter;
+import io.opentelemetry.sdk.metrics.internal.aggregator.Aggregator;
+import io.opentelemetry.sdk.metrics.internal.descriptor.MetricDescriptor;
+import io.opentelemetry.sdk.resources.Resource;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/** Aggregation that selects the specified default based on insturment. */
+class DefaultAggregation extends Aggregation {
+
+  static final Aggregation INSTANCE = new DefaultAggregation();
+
+  private static final ThrottlingLogger logger =
+      new ThrottlingLogger(Logger.getLogger(DefaultAggregation.class.getName()));
+
+  private DefaultAggregation() {}
+
+  private static Aggregation resolve(InstrumentDescriptor instrument) {
+    switch (instrument.getType()) {
+      case COUNTER:
+      case UP_DOWN_COUNTER:
+      case OBSERVABLE_SUM:
+      case OBSERVABLE_UP_DOWN_SUM:
+        return SumAggregation.DEFAULT;
+      case HISTOGRAM:
+        return ExplicitBucketHistogramAggregation.DEFAULT;
+      case OBSERVABLE_GAUGE:
+        return LastValueAggregation.INSTANCE;
+    }
+    logger.log(Level.WARNING, "Unable to find default aggregation for instrument: " + instrument);
+    return NoAggregation.INSTANCE;
+  }
+
+  @Override
+  public <T> Aggregator<T> createAggregator(
+      Resource resource,
+      InstrumentationLibraryInfo instrumentationLibraryInfo,
+      InstrumentDescriptor instrumentDescriptor,
+      MetricDescriptor metricDescriptor,
+      ExemplarFilter exemplarFilter) {
+    return resolve(instrumentDescriptor)
+        .createAggregator(
+            resource,
+            instrumentationLibraryInfo,
+            instrumentDescriptor,
+            metricDescriptor,
+            exemplarFilter);
+  }
+
+  @Override
+  public String toString() {
+    return "default";
```

Comment:
might be nice to have a little be more here.  Is this really better than the default `toString()` implementation?

Reply:
I tend to do this for my own sanity in tests when doing diffs/toStrings.   Whatever makes the most sense, I can change it to defaultAggregation.  For context it's because the string was "aggregation: ${aggregation}" or something like that.

Reply:
yep. Just like to have a little bit more context in these so you can see the type of the thing if it's embedded in another object of some kind.

Reply:
Good point.  Fixed.